backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/document/impl/DocTypeDAO.java

```diff
@@ -343,4 +346,20 @@ public Optional<String> getCounterDocBaseTypeByDocBaseType(@NonNull final String
 			return Optional.ofNullable(counterDocBaseTypeByDocBaseType.get(docBaseType));
 		}
 	}
+
+	@Cached
+	@Override
+	public DocType loadById(final DocTypeId id)
```

Comment:
* tomato tomato: why not calling it getById like everywhere else and rename the existing getById method to getRecordById
* avoid using `@Cached` because it does not have a great cache invalidation support. Consider using an CCache<DocTypeId, DocType>.
    * NOTE: `@Cached` won't know to invalidate the cache in case somebody is changing a document type

Reply:
removed @Cached and renamed to taste.

Question: should we then deprecate `@Cached`?